### ISSUE TITLE
List of the 10 training fonts?

### ISSUE THREAD

**Issue by koch-aai** [Shreeshrii/tessdata_shreetest#1]
Thanks for uploading this trained model - could you possibly provide some info about the training data? 

Specifically the fonts used and the average string length. Has this been tested on SVHN by any chance?

Thanks!

**Comment by Shreeshrii** [comment]
1. It has NOT been tested at all. It is a proof of concept finetune training. Users are encouraged to finetune specific to their own user case, fonts etc.

2. The training text used for last training version with commas is at 
https://github.com/Shreeshrii/tessdata_shreetest/blob/master/eng.digits.training_text

3. Modified versions of eng.punc and eng.numbers have been used. These could be further modified based on user requirements. They might cause minor improvements in recognition. Files used  can be compared with the ones in langdata/eng and are made available at 
https://github.com/Shreeshrii/tessdata_shreetest/blob/master/eng.digits.punc
https://github.com/Shreeshrii/tessdata_shreetest/blob/master/eng.digits.numbers

4. Fonts used were those listed in src/training/language_specific.sh for Latin script (with minor modifications).  Again, finetuning with fonts used in images to be OCRed will lead to better accuracy.
Here is the list used

```
./digits/eng.Arial_Bold.exp0.lstmf
./digits/eng.Arial_Bold_Italic.exp0.lstmf
./digits/eng.Arial.exp0.lstmf
./digits/eng.Arial_Italic.exp0.lstmf
./digits/eng.Courier_New_Bold.exp0.lstmf
./digits/eng.Courier_New_Bold_Italic.exp0.lstmf
./digits/eng.Courier_New.exp0.lstmf
./digits/eng.Courier_New_Italic.exp0.lstmf
./digits/eng.FreeMono.exp0.lstmf
./digits/eng.FreeSans.exp0.lstmf
./digits/eng.FreeSerif.exp0.lstmf
./digits/eng.Georgia_Bold.exp0.lstmf
./digits/eng.Georgia_Bold_Italic.exp0.lstmf
./digits/eng.Georgia.exp0.lstmf
./digits/eng.Georgia_Italic.exp0.lstmf
./digits/eng.Times_New_Roman_Bold.exp0.lstmf
./digits/eng.Times_New_Roman_Bold_Italic.exp0.lstmf
./digits/eng.Times_New_Roman.exp0.lstmf
./digits/eng.Times_New_Roman_Italic.exp0.lstmf
./digits/eng.Trebuchet_MS_Bold.exp0.lstmf
./digits/eng.Trebuchet_MS_Bold_Italic.exp0.lstmf
./digits/eng.Trebuchet_MS.exp0.lstmf
./digits/eng.Trebuchet_MS_Italic.exp0.lstmf
./digits/eng.Verdana_Bold.exp0.lstmf
./digits/eng.Verdana_Bold_Italic.exp0.lstmf
./digits/eng.Verdana.exp0.lstmf
./digits/eng.Verdana_Italic.exp0.lstmf
```

**Comment by Bech007** [comment]
Hi @Shreeshrii  
i have a dataset .txt i want train tesseract at them but i don't know how i can do that?
thank

**Comment by Shreeshrii** [comment]
Please see https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract-4.00
and
https://github.com/Shreeshrii/tess4training

**Comment by myjun1124** [comment]
This digit traindata was very useful to me. It worked well after preprocessing that I used for recognizing temperature on the screen from thermal camera.

**Comment by myjun1124** [comment]
This digit traindata was useful to me. It worked well after preprocessing that I used for recognizing temperature on the screen from thermal camera.

**Comment by Shreeshrii** [comment]
Thanks for the comment @myjun1124. Glad to know it worked for you.

**Comment by nikhilcms** [comment]
Hi @Shreeshrii , I found that tesseract 4.1.1 works good for extraction of words, but many times in failed to extract digits ( specifically bold ),  how can i solve this issue ?
